exporter/README.md

```diff
@@ -0,0 +1,4 @@
+Apache Exporter for Prometheus
+============
+
+Prometheus exporter for Apache server metrics is hosted at https://github.com/Lusitaniae/apache_exporter.
```

Comment:
Two sentences about port/usage would be useful. Or a pointer to an examples in tests.

Reply:
Discussed offline - I'll keep changes as they are for now.